src/map/clif.c

```diff
@@ -11835,7 +11835,7 @@ static void clif_parse_WalkToXY(int fd, struct map_session_data *sd)
 
 	// Do not allow one cell move commands if the target cell is not free
 	if (battle_config.official_cell_stack_limit > 0
-		&& check_distance_blxy(&sd->bl, x, y, 1) && map->count_oncell(sd->bl.m, x, y, BL_CHAR | BL_NPC, 1))
+		&& check_distance_blxy(&sd->bl, x, y, 1) && map->count_oncell(sd->bl.m, x, y, BL_CHAR | BL_NPC, 2))
```

Comment:
Is the removal of the `0x1` bit (meant to skip units that are in transit) intentional here?

Reply:
It was meant to skip those units under `sc->option&OPTION_INVISIBLE`, like characters using `@hide`, and other invisible units that players should not be aware of.

Reply:
But since it uses `official_stack_limit`, shouldn't it follow its mechanics? We always ignore moving units in those checks (on a side note, why does this enforce a count of 0 regardless of the value of official_stack_limit? i.e. `official_stack_limit = 5` would consider a cell with 4 unit stacked as still available to walk to)

Reply:
Makes sense. I've updated the PR with that taken into consideration. Thanks!